### ISSUE TITLE
Remember background-player playlist after application restart or system reboot

### ISSUE THREAD

**Issue by mafrailea** [TeamNewPipe/NewPipe#4299]
#### Describe the feature you want

I use NewPipe as a music player. I've created several public playlists in Youtube that I launch in the background player.

It would be nice if NewPipe could remember the playlist and track that was being played after an application restart or system reboot.

#### Is your feature request related to a problem? Please describe it

The main issue is that, each time the system kills the application or I reboot the system, NewPipe doesn't recall the playlist / track which is being player.

#### Additional context

I think it would make sense to implement something like this:
- Each time NewPipe changes to a new track in the playlist, save the current playlist and current track information to persistent storage.
- Implement a way (maybe left menu) to launch the background player.

**Comment by opusforlife2** [comment]
Why not save them as local playlists?

**Comment by mafrailea** [comment]
> Why not save them as local playlists?
I do it, but it doesn't remember the current track.

When you have a playlist with +400 tracks, it's hard to remember which track you were playing and it's not easy to locate that track on the list.

**Comment by opusforlife2** [comment]
Oh alright. So you would like the option of having Newpipe remember the 'last played' state for playlists and tracks, kind of like podcast apps do.

**Comment by mafrailea** [comment]
> Oh alright. So you would like the option of having Newpipe remember the 'last played' state for playlists and tracks, kind of like podcast apps do.

Right, specifically what is being played on the background player. :)

**Comment by opusforlife2** [comment]
They're all the same player next release onwards. ;)

**Comment by MD77MD** [comment]
l like this idea alot ... but my usage scenar is a bit different.. i would use it to go through my watch later or shows playlists.

however, I have an even better idea. it would also be great if we can have the last video played from each playlist highlighted. the whenever you open a playlist (not play) the screen would snap to that video (like in mx player)...  

I think this would go great with @mafrailea idea and save us the constant hunt on where did we stop. what do you think @opusforlife2

**Comment by opusforlife2** [comment]
You don't need to solicit my opinion. I watch every issue that is opened in this repo. If I have something to say, I will have already said it. :P

**Comment by MD77MD** [comment]
(solicit) i don't really know this word but I think you misunderstood me...  the truth is I usually like to hear your opinion on any idea i suggest... perhaps because your like me regarding paying attention to the small details or maybe because your input is usually more positive ... so i like to talk to you more 😊

**Comment by opusforlife2** [comment]
Solicit = ask for. I mean you don't need to personally ask for my opinion, because I usually give it on my own when I have something I want to say. 🚀

**Comment by MD77MD** [comment]
ok from now on i will tag you when i really need your input... hopefully not so much but no promises 😁
